new_event.md

```diff
@@ -229,20 +231,14 @@ Remember to allow for daylight savings as necessary.
 '{"HW": "Hardware", "GAM": "Games"}'
 ```
 
-**Questions** is an array of string keys the front end uses to show the correct question to the users. The keys and their related questions are in `client/src/project/CustomQuestions.js` e.g.
+**Questions** is an array of string keys the front end uses to show the correct question to the users. The keys and their related questions are in [client/src/project/CustomQuestions.js](https://github.com/CoderDojo/coolest-platform/blob/master/client/src/project/CustomQuestions.js) e.g.
 
 ```
 '["travel_bursary", "attendance_prev_year"]'
 ```
 
-**tx** is the timezone, the frontend displays this so it needs to be correct. For UK it will be `'Europe/London'`, for the international event in Dublin it will be `'Europe/Dublin'`. The timezone for the US event will need to be provided or found.
+**tz** is the timezone, the frontend displays this so it needs to be correct. For UK it will be `'Europe/London'`, for the international event in Dublin it will be `'Europe/Dublin'`. The timezone for the US event will need to be provided or found.
 
 ---
 
-Update script to just create empty event with id,
-
-Then create update script to set values correctly
-
-script in readme, add variables from google doc
-
-update kube.sh to fix event_slugs
+Once the event is populated with the correct information you should be able to navigate to the new site with the event slug, and register a project.
```

Comment:
Didn't cop originally that these were notes for yourself to update in the docs. We still do need the docs through for updating the event slugs in kube.sh.